src/AudioToolbox/MusicTrack.cs

```diff
@@ -263,7 +263,15 @@ public MusicPlayerStatus SetDestMidiEndpoint (MidiEndpoint endpoint)
 		{
 			return MusicTrackSetDestMIDIEndpoint (handle, endpoint == null ? MidiObject.InvalidRef : endpoint.MidiHandle);
 		}
-#endif // IOS
+
+		[DllImport (Constants.AudioToolboxLibrary)]
+		extern static /* OSStatus */ MusicPlayerStatus MusicTrackGetDestMIDIEndpoint (/* MusicTrack */ IntPtr inTrack, MidiEndpointRef outEndpoint);
+
+		public MusicPlayerStatus GetDestMIDIEndpoint (MidiEndpoint endpoint)
```

Comment:
I think you want to:

`GetDestMIDIEndpoint` => `GetDestMidiEndpoint`

Reply:
missed that one, will update as soon as I get more reviews